wegodang/users/views.py

```diff
@@ -1,3 +1,64 @@
-from django.shortcuts import render
+import jwt
+import requests
+import json
 
-# Create your views here.
\ No newline at end of file
+from django.http            import JsonResponse
+from django.core.exceptions import ValidationError
+from django.views           import View
+from django.conf            import settings
+
+from users.models           import User
+from users.validator        import validate_email
+
+class KaKaoSignUpView(View):
+    def get(self, request):
+        access_token     = request.headers.get('Authorization')
+        KAKAO_INFO_API   = 'https://kapi.kakao.com/v2/user/me'
+        response         = requests.get(KAKAO_INFO_API, headers={'Authorization': f'Bearer {access_token}'}, timeout=5)
+
+        if not response.status_code == 200:
+            return JsonResponse({'message' : 'INVALID_RESPONSE' }, status = 400)
+
+        user_information = response.json()
+        kakao_id         = user_information['id']
+        user_name        = user_information['properties']['nickname']
+        email            = user_information['kakao_account']['email']
+        is_user          = False
+
+        if User.objects.filter(kakao_id = kakao_id).exists():
+            is_user = True
```

Comment:
User.objects.filter(kakao_id = kakao_id).exists():의 결과 자체가 boolean이기 때문에 if는 필요 없을 것 같습니다.
```suggestion
        is_user          = User.objects.filter(kakao_id = kakao_id).exists():

```

Reply:
코드 수정하였습니다 !